### PR TITLE
Adding a new S3 bucket for SAML metadata in LF CFN

### PR DESCRIPTION
**Pull request by smunigatiaws**
Added a new S3 bucket for SAML metadata and modified the templates to reflect the changes.

### CODE REVIEW

File: resources/code/lf-emr-workshop-js.template [hunk 1]
```diff
@@ -547,7 +547,7 @@
           "Fn::Join": [
             "-",
             [
-              "notebooks",
+              "lf-notebooks-bucket",
```
Comment (emrspecialistsamer) [inline resources/code/lf-emr-workshop-js.template]: Please use lf-scripts-bucket as discussed

File: resources/code/lf-workshop-js.template [hunk 2]
```diff
@@ -838,6 +716,22 @@
...
             }
         },
+        "MetadataBucket": {
+            "Type": "AWS::S3::Bucket",
+            "Properties": {
+                "BucketName": {
+                    "Fn::Join": [
+                        "-",
+                        [
+                            "lf-metadata-bucket",
```
Comment (emrspecialistsamer) [inline resources/code/lf-workshop-js.template]: As discussed, we do not want multiple buckets. please use lf-scripts-bucket-<acct-id> and have proper prefixes for the metadata and notebooks. I am ok for now but want this to be fixed after next week (when you add your ADFS changes)

File: resources/code/scripts/LF-EMR-Jupyter.ipynb [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "org.apache.spark.sql.AnalysisException: org.apache.hadoop.hive.ql.metadata.HiveException: Unable to fetch table dl_tpc_item. Insufficient Lake Formation permission(s) on dl_tpc_item (Service: AWSGlue; Status Code: 400; Error Code: AccessDeniedException; Request ID: 52cf5f90-df6f-4a08-9cb5-ccf701c06f8c);\n",
```
Comment (emrspecialistsamer) [inline resources/code/scripts/LF-EMR-Jupyter.ipynb]: May be you can clear the notebook output?

File: website/content/90-cleanups/_index.md [hunk 4]
```diff
@@ -6,5 +6,5 @@ chapter = true
 <center><h2>Cleaning Up</h2></center>
 
 <div style="text-align: justify">
-When you have finished with this workshop, remember to clean up all those AWS resources that you created using AWS CloudFormation. Use the AWS CloudFormation console or AWS CLI to delete the stack named <b>Lake-Formation-WF</b> and <b>Lake-Formation-With-EMR-WF</b>.
+When you have finished with this workshop, remember to clean up all those AWS resources that you created using AWS CloudFormation. You have to delete the content in the S3 buckets created part of this exercise before deleting the CloudFormation Stacks. Use the AWS CloudFormation console or AWS CLI to delete the stack named <b>Lake-Formation-WF</b> and <b>Lake-Formation-With-EMR-WF</b>.
```
Comment (emrspecialistsamer) [inline website/content/90-cleanups/_index.md]: Provide a link on emptying the folder and then deleting the S3 bucket

File: resources/code/lf-workshop-js.template (hunk 2 above)
Comment (emrspecialistsamer) [reply resources/code/lf-workshop-js.template]: Also remove the word "bucket" from the bucket name

Comment (emrspecialistsamer) [reply resources/code/lf-workshop-js.template]: not sure why it was used even with datalake bucket

File: resources/code/lf-emr-workshop-js.template (hunk 1 above)
Comment (smunigatiaws) [reply resources/code/lf-emr-workshop-js.template]: Sure will re-factor this bucket creation after the workshop next week.

File: resources/code/lf-workshop-js.template (hunk 2 above)
Comment (smunigatiaws) [reply resources/code/lf-workshop-js.template]: Sure will re-factor this bucket creation after the workshop next week.

File: resources/code/scripts/LF-EMR-Jupyter.ipynb (hunk 3 above)
Comment (smunigatiaws) [reply resources/code/scripts/LF-EMR-Jupyter.ipynb]: The reason I left it output  error message there to show even if they have issues running the query or some environment issues etc.

File: website/content/90-cleanups/_index.md (hunk 4 above)
Comment (smunigatiaws) [reply website/content/90-cleanups/_index.md]: Provided more instructions and added screen to support this activity.

File: resources/code/scripts/LF-EMR-Jupyter.ipynb (hunk 3 above)
Comment (emrspecialistsamer) [reply resources/code/scripts/LF-EMR-Jupyter.ipynb]: Alright.

File: website/content/90-cleanups/_index.md (hunk 4 above)
Comment (emrspecialistsamer) [reply website/content/90-cleanups/_index.md]: CFN delete will remove the resources (the first sentence is not right). The problem is only with S3 content (and VPCs sometimes when they have ENIs attached) as CFN wont remove the bucket.

**Pull request merged**